core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/PersonAccountEditViewModel.kt

```diff
@@ -0,0 +1,19 @@
+package com.ustadmobile.core.viewmodel
+
+data class PersonUsernameAndPasswordModel(
+
+    val username: String = "",
+
+    val currentPassword: String = "",
+
+    val newPassword: String = "",
+
+    val passwordConfirmed: String = "",
+)
+
+data class PersonAccountEditUiState(
+
+    val personUsernameAndPassword: PersonUsernameAndPasswordModel = PersonUsernameAndPasswordModel(),
+
+    val currentPasswordError: String = "",
+)
```

Comment:
Please add a boolean for fieldsEnabled. We should use this on all screens where the user is editing something. FieldsEnabled will normally be true, but will be set to false when loading, saving etc.